packages/server-wallet/src/models/channel.ts

```diff
@@ -173,28 +175,21 @@ export class Channel extends Model implements ChannelColumns {
     await Channel.query(txOrKnex).findOne({channelId}).patch({initialSupport: support});
   }
 
-  static async setLedger(
-    channelId: Bytes32,
-    assetHolderAddress: Address,
-    txOrKnex: TransactionOrKnex
-  ): Promise<void> {
-    await Channel.query(txOrKnex).findOne({channelId}).patch({assetHolderAddress});
+  static async setLedger(channelId: Bytes32, txOrKnex: TransactionOrKnex): Promise<void> {
+    await Channel.query(txOrKnex).findOne({channelId}).patch({isLedgerChannel: true});
```

Comment:
aside: this function is _only ever used during testing_, and seems like it doesn't belong in production code